DrugResistance/src/main/java/edu/stanford/hivdb/drugresistance/database/ConditionalComments.java

```diff
@@ -146,8 +146,8 @@ public String getDrugLevelsText() {
 		final private String comment;
 		final private Collection<String> highlightText;
 		final private Mutation mutation;
-
-		private BoundComment(
+		
+		protected BoundComment(
```

Comment:
Same rationale as comment above. Can also change back private and test in terms of public methods if it's preferred.